### DISCUSSION TITLE
有 Kotlin/Native 的需求吗?

### DISCUSSION THREAD

**Discussion by ntbps** [Tencent/MMKV#589] [Ideas]
RT

**Comment by lingol** [comment]
What do you mean? MMKV does support Kotlin.

**Reply by KunMinX** [reply]
His point is, is there a plan to support pure Kotlin desktop development? The existing MMKV and Android APIs such as Context and ContentProvider are tightly coupled and cannot be used directly on desktop. Compose Desktop supports Windows, MacOS, and Linux, so we can't simply use Context and ContentProvider.

**Comment by lingol** [comment]
Good news. We have just added KMM experiment support in the dev branch. You can try it out.

**Comment by lingol** [comment]
It's supported in [v2.4.1](https://github.com/Tencent/MMKV/releases/tag/v2.4.1).
